vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -62,6 +63,7 @@ var (
 	webhookPort     = flag.String("webhook-port", "", "Server Port for Webhook")
 	registerWebhook = flag.Bool("register-webhook", true, "If set to true, admission webhook object will be created on start up to register with the API server.")
 	registerByURL   = flag.Bool("register-by-url", false, "If set to true, admission webhook will be registered by URL (webhookAddress:webhookPort) instead of by service name")
+	vpaNamespace    = flag.String("namespace", core.NamespaceAll, "Namespace to search for VPA objects. Empty means all namespaces will be used.")
```

Comment:
maybe "vpa-object-namespace" to not confuse with namespace for the pod to run in.